cc_midterm/cc_midterm/templates/base.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <nav class="menu">
+        <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="/signup">Sign Up</a></li>
+            <li><a href="/login">Login</a></li>
+            <li><a href="/logout">Logout</a></li>
+        </ul>
```

Comment:
I would personally just do this with a bunch of `<span>` instead of `<ul>`/`<li>`. This would put them in one row so it takes up less space.

Haven't tested, but potentially something like:
```html
        <nav class="menu">
            <span><a href="/">Home</a></span>
            <span><a href="/signup">Sign Up</a></span>
            <span><a href="/login">Login</a></span>
            <span><a href="/logout">Logout</a></span>
            <hr/>
        </nav>
```